mod_app/admin/link_admin.py

```diff
@@ -1,7 +1,78 @@
 from django.contrib import admin
-from mod_app.utils.mixins import PreviewMixin, s3BrowserButtonMixin
 
-from mod_app.models import *
+from mod_app.models.film_model import Film
+from mod_app.models.support_models import (
+    CardImage,
+    Drawing,
+    Keyword,
+    OtherLink,
+    Postcard,
+    Poster,
+    PressBook,
+    Programme,
+    Publicity,
+    PublicVisualInfluence,
+    Script,
+    Source,
+    Still,
+    Tag,
+    Topic,
+    Video,
+)
+from mod_app.admin.utils import register_custom_admin, custom_inline
+from mod_app.utils.mixins import (
+    PreviewMixin,
+    s3BrowserButtonMixin,
+)
+
+COMMON_FILELINK_CLASSES = [
+    Script,
+    PressBook,
+    Programme,
+    Publicity,
+    Still,
+    Postcard,
+    Poster,
+    Drawing,
+    PublicVisualInfluence,
+]
+
+COMMON_FILELINK_CLASS_INLINES = [
+    custom_inline(
+        model,
+        mixins=(
+            PreviewMixin,
+            s3BrowserButtonMixin,
+        ),
+        options={
+            "extra": 1,
+            "classes": [
+                "inline-inline",
+                "grp-collapse",
+                "grp-closed",
+            ],
+            "autocomplete_fields": ("archive",),
```

Comment:
ok after looking into when we use tuples and lists in the admin, it seems this field is actually agnostic, so if you'd prefer to use list notation for consistency then that's fine with me